test/e2e/tests/switch-custom-network.spec.js

```diff
@@ -88,7 +88,7 @@ describe('Switch ethereum chain', function () {
         await driver.switchToWindow(extension);
 
         const currentNetworkName = await driver.findElement({
-          tag: 'p',
+          tag: 'span',
```

Comment:
Changed the wrapping html element of the text from a `p` to `span` tag in the `PickerNetwork`